provider/smsc/smsc.go

```diff
@@ -0,0 +1,63 @@
+package smsc
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+	"github.com/messagebird/sachet"
+)
+
+type SmscConfig struct {
+	Login    string `yaml:"login"`
+	Password string `yaml:"password"`
+	Sender   string `yaml:"sender"`
+}
+
+const SmscRequestTimeout = time.Second * 60
+
+type Smsc struct {
+	Login    string
+	Password string
+        Sender   string
```

Comment:
be sure you run `go fmt` before committing the code (https://blog.golang.org/go-fmt-your-code)

Reply:
my bad, fixed in 919813f